cmake.deps/cmake/BuildLuarocks.cmake

```diff
@@ -192,7 +192,16 @@ if(USE_BUNDLED_BUSTED)
   # luv
   set(LUV_DEPS luacheck)
   if(USE_BUNDLED_LUV)
-    list(APPEND LUV_DEPS luv-static lua-compat-5.3)
+
+    # HACK: the luv rockspec requires libuv.a or libuv.so
+    add_custom_target(link-libuv
+      COMMAND ${CMAKE_COMMAND} -E create_symlink
+        ${DEPS_INSTALL_DIR}/lib/libuv_a.a
+        ${DEPS_INSTALL_DIR}/lib/libuv.a
```

Comment:
Just move it maybe? Dealing with all these other things that don't know to look for `uv_a` seems like a lot more work.

Reply:
This works ok for unix. The issue is the library has a different path for windows.

Reply:
Windows is the worst